src/logic/utils.ts

```diff
@@ -1,14 +1,28 @@
 import {
     NEW_PLAYER_HIGHLIGHT_DAYS,
+    NUMBER_OF_PLAYERS_FOR_VALID_MATCH,
+    PLAYER_MAXIMUM_GAMES_AS_NEW_PLAYER,
     PLAYER_MINIMUM_GAMES_REQUIRED,
     PLAYER_MINIMUM_WINS_REQUIRED
 } from "../components/constants";
 import { Player } from "../types/domain/Player";
 
+
+/**
+ * Gets the win rate as a percentage (number, without the "%").
+ * @param winCount Wins - most likely valid wins.
+ * @param totalCount Number of matches to consider - most likely valid matches.
+ * @returns A whole number from 0 to 100.
```

Comment:
In this case, your description of a "a whole number" is a bit too literal :D we want to let the user know what the his number represents. In this case, 

the win percentage as represented as number from  0 to 100.

Reply:
but the number _is_ rounded. it is a whole number as opposed to a decimal. why not state that? doesn't cost much

Reply:
changed to integer, thank you!